content/en/docs/languages/js/getting-started/nodejs.md

```diff
@@ -24,6 +24,10 @@ Ensure that you have the following installed locally:
 - [TypeScript](https://www.typescriptlang.org/download), if you will be using
   TypeScript.
 
+{{% alert title="Note" color="info" %}}
+Much of OpenTelemetry JS documentation is written assuming the compiled application is run as CommonJS.
```

Comment:
```suggestion
The OpenTelemetry documentation assumes that the compiled application is run as CommonJS.
```

Would it make sense to add a link to CommonJS? Are all Node developers familiar with it?